awx/main/utils/filters.py

```diff
@@ -83,39 +83,41 @@ def filter(self, record):
         False - should not be logged
         True - should be logged
         """
-        # Do not send exceptions to external logger
+        # Do not send logging exceptions to external logger
         if record_is_blocked(record):
             return False
+
         # General enablement
         if not self.enabled_flag:
             return False
 
-        # Logger type enablement
-        loggers = self.enabled_loggers
-        if not loggers:
-            return False
-        if record.name.startswith('awx.analytics'):
-            base_path, headline_name = record.name.rsplit('.', 1)
-            return bool(headline_name in loggers)
-        else:
-            if '.' in record.name:
-                base_name, trailing_path = record.name.split('.', 1)
-            else:
-                base_name = record.name
-            return bool(base_name in loggers)
```

Comment:
Essentially this is moved from `ExternalLoggerEnabled` to `DynamicLevelFilter`, but we keep the "awx" logger check here as an exception specific to the external logger.